Wabbajack.Common/Json.cs

```diff
@@ -72,7 +72,7 @@ public static T FromJSON<T>(this Stream data)
             try
             {
                 return JsonConvert.DeserializeObject<T>(s,
-                    new JsonSerializerSettings {TypeNameHandling = TypeNameHandling.Auto, Converters = Converters});
+                    new JsonSerializerSettings {TypeNameHandling = TypeNameHandling.Auto, Converters = Converters})!;
```

Comment:
**Example of overriding a null warning/error and pushing it through anyway**
I -think- the json API is bad, in that it returns a nullable object, but it would never do so.  It would either return an object, or throw an exception.  But the fact that it returns a nullable object makes the compiler complain when we try to return it in a non-nullable value.  We can override its complaining with `!`